configs.example/datamodule/streamed_batches.yaml

```diff
@@ -1,6 +1,8 @@
 _target_: pvnet.data.datamodule.DataModule
 # Path to the data configuration yaml file. You can find examples in the configuration subdirectory
 # in configs.example/datamodule/configuration
+# Use the full local path such as: /FULL/PATH/PVNet/configs/datamodule/configuration/gcp_configuration.yaml"
+num_workers: 20
```

Comment:
pre-commit is failing on this right now, with duplicate keys, so might need to remove it
```suggestion
```